plugin/trino-delta-lake/src/test/resources/databricks153/type_widening/README.md

```diff
@@ -0,0 +1,67 @@
+Data generated using Databricks 15.3:
+
+```sql
+--      * byte	    -> short, int, long, decimal, double
+--      * short	    -> int, long, decimal, double
+--      * int	    -> long, decimal, double
+--      * long	    -> decimal
+--      * float	    -> double
+--      * decimal	-> decimal with greater precision and scale
+--      * date	    -> timestampNTZ
+--
+--  Covering all possible type changes
+--  from https://docs.databricks.com/en/delta/type-widening.html#supported-type-changes
+
+CREATE TABLE default.test_db_type_widening
+    (
+        bts BYTE, bti BYTE, btl BYTE, btd BYTE, btdb BYTE,
```

Comment:
`byte_to_short` seems more readable than `bts`

Reply:
Agree, what was I thinking? I will rename these in a separate push.